src/communication/scrollsguide.py

```diff
@@ -4,10 +4,29 @@
 import aiohttp
 import json
 from urllib.parse import quote_plus
+from typing import Any, Dict, Tuple, Type
+from fuzzywuzzy import fuzz
 
+class ScrollNotFound(Exception):
+    pass
+
+
+class MultipleScrollsFound(Exception):
+    def __init__(self, scrolls, search_term):
+        self.scrolls = [scroll.name for scroll in scrolls]
+        self.search_term = search_term
+
+    def __str__(self):
+        if len(self.scrolls) > 5:
+            return f"Too many scrolls start with {self.search_term}"
+        return f'Multiple scrolls starting with {self.search_term}: {", ".join(self.scrolls)}'
 
 class Scroll:
     """Wrapper class for Scroll information coming from the API"""
+
+    _scrolls_db = None
+    _scrolls_names = None
```

Comment:
Do we need both? If you just keep names and put entire Scroll in there, it shouldn't cause much troubles right?

Reply:
We could. But what if we need to store scroll id somewhere and then by id. But sure, fair enough